models/tensorswap/solana/tensorswap_solana_trades_schema.yml

```diff
@@ -0,0 +1,171 @@
+version: 2
+
+models:
+  - name: tensorswap_solana_trades
+    meta:
+      blockchain: solana
+      project: tensor
+      contributors: [ilemi]
+    config:
+      tags: ['tensorswap','solana','nft']
+    description: >
+       tensorswap solana trades
+    tests:
+     - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - project
+            - trade_category
+            - outer_instruction_index
+            - inner_instruction_index
+            - account_metadata
+            - tx_id
+     - check_seed:
+          seed_file: ref('tensorswap_solana_base_trades_seed')
+          match_columns:
+            - block_slot
+            - tx_id
+            - outer_instruction_index
+            - inner_instruction_index
+          check_columns:
+            - buyer
+            - seller
+            - account_mint
+            - amount_raw
+            - taker_fee_amount_raw
+            - maker_fee_amount_raw
+            - royalty_fee_amount_raw
+    columns:
+     - &blockchain
+       name: blockchain
+       description: "Blockchain"
+     - &project
+       name: project
+       description: "Project"
+     - &version
+       name: version
+       description: "Project version"
+     - &block_time
+       name: block_time
+       description: "UTC event block time"
+     - &token_name
+       name: token_name
+     - &token_symbol
+       name: token_symbol
+     - &trade_type
+       name: trade_type
+       description: "primary or secondary"
+     - &number_of_items
+       name: number_of_items
+       description: "Number of items traded"
+     - &trader_category
+       name: trade_category
+       description: "buy or sell"
+     - &seller
+       name: seller
+       description: "Seller wallet address"
+     - &buyer
+       name: buyer
+       description: "Buyer wallet address"
+     - &amount_usd
+       name: amount_usd
+       description: "USD value of the trade at time of execution"
+       tests:
+         - dbt_utils.accepted_range:
+             max_value: 1000000000 # $1b is an arbitrary number, intended to flag outlier amounts early
```

Comment:
similar idea here, can likely remove, especially if we remove join to prices at project-level too